magellan-library/src/main/java/com/wealthfront/magellan/core/Displayable.kt

```diff
@@ -6,6 +6,8 @@ public interface Displayable {
 
   public val view: View?
 
+  public var templatedView: View?
```

Comment:
storing any generated templatedView on the Displayable, to preserve encapsulation. I suppose it could also work to store a map at the navigator level (i.e. Displayable -> It's templated View, if there is one). But I had a couple of worries about that: 
- Could be harder to ensure the map gets cleaned up as needed when Displayable are torn down. I guess we'd do this inside navigateTo/navigateFrom instead of in response to onHide events? Feels more brittle.
- Need to guarantee unique keys for this map. However, it's currently enforced that a given Navigable only appears once in a given backstack so this likely isn't a problem.

Reply:
Possibly dumb question, is there a way to include an `internal` property in an interface? I'd really rather not expose `templatedView` property to developers working inside a Screen/Step etc.